### PR TITLE
asg hist integration

### PR DESCRIPTION
**Pull request by SidelineCory24**
[Original PRs](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2023)

- [x] Integrate with the new Assignment endpoint
- [x] Add status to assignment history for agenda item (Eg. EF, CP, AP)
- [x] Add additional information as appropriate (TOD description has been added as well)
- [x] Verify Position Details/History on client view remains the same

[BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/668)

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Client Profile unchanged
image

Assignments Hist Tab with `Status` and `TOD Description`
image

File: src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx [hunk 1]
```diff
@@ -49,9 +49,12 @@ const AgendaItemResearchPane = props => {
   const [selectedNav, setSelectedNav] = useState(get(tabs, '[0].value') || '');
 
   // assignments
-  const { data, error, loading /* , retry */ } = useDataLoader(api().get, `/fsbid/client/${perdet}/`);
-  const assignments = get(data, 'data.assignments') || [];
-  const languages = get(data, 'data.languages') || [];
+  // need to update once fully integrated
+  const { data, error, loading } = useDataLoader(api().get, `/fsbid/assignment_history/${perdet}/`);
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:53]: u can keep in the commented out `retry`

Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:53]: done

File: src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx [hunk 2]
```diff
@@ -33,6 +36,18 @@ const AssignmentsContent = ({ assignment }) => (
       <span className="bid-list-card-title-post">Language: </span>
       {get(assignment, 'position.language', NO_LANGUAGES)}
     </div>
+    { has(assignment, 'status') &&
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/Assignments/AssignmentsListResultsCard/AssignmentsContent/AssignmentsContent.jsx:39]: 🎊

**Review by elizabeth-jimenez — APPROVED** [review]

**Comment by elizabeth-jimenez** [comment]
Yay!

**Pull request merged**